lib/order.rb

```diff
@@ -0,0 +1,28 @@
+class Order
+  attr_reader :foods
+
+  def initialize(menu)
```

Comment:
I believe this is initializing with a new instance of the menu class right? If so, you could have used a default argument here so you don't need to pass an argument everytime.